purchase_order_webkit_with_notes/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=====================================
+Purchase Order Qweb Report With Notes	
+=====================================
+
+* This module replaces the webkit Quotation / Purchase Order report by
+  a qweb report and adds header and footer notes on the purchase order 
+  and quotation
+
+
+Installation
+============
+
+To install this module there is no external library required.
+
+1. Nothing required at all.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+1. No Configuration required at all.
+
+Usage
+=====
+
+* This module will be helpful to have the notes in header and footer for any Purchase Order.
```

Comment:
This should be in introduction not in usage.
Indicate here where you can find the report or add a screenshot